chef_master/source/resource_remote_file.rst

```diff
@@ -600,6 +619,58 @@ To prevent the chef-client from re-downloading files that are already present on
 
 The desired approach just depends on the desired workflow. For example, if a node requires a new file every day, using the checksum approach would require that the local checksum be updated and/or verified every day as well, in order to ensure that the local checksum was the correct one. Using a conditional ``GET`` in this scenario will greatly simplify the management required to ensure files are being updated accurately.
 
+Access a remote UNC path on Windows
+-----------------------------------------------------
+The ``remote_file`` resource now supports the use of credentials on Windows when accessing a remote UNC path on Windows such as ``\\myserver\myshare\mydirectory\myfile.txt``. This
+allows access to the file at that path location even if the Chef client process identity does not have permission to access the file. The new properties ``remote_user``, ``remote_domain``, and ``remote_password`` may be used to specify credentials with access to the remote file so that it may be read. Please check the details of the usage of these properties above.
+
+**Note**: This feature is mainly used for accessing files between two nodes in different domains and having different user accounts.
```

Comment:
This is primarily for accessing remote files when the user that the Chef client is running as does not have sufficient access and alternate credentials need to be specified. If the user already has access, the credentials do not need to be specified.